docs/cli.md

```diff
@@ -286,17 +286,106 @@ viam organizations --help
+viam dataset data remove ids --dataset-id=123 --org-id=123 --location-id=123 --file-ids=123,456
+```
+
+#### Command options
+
+<!-- prettier-ignore -->
+| Command option | Description | Positional arguments |
+| -------------- | ----------- | -------------------- |
+| `create` | Create a new dataset | - |
+| `rename` | Rename an existing dataset | - |
+| `list` | List dataset information from specified IDs or for an org ID | - |
+| `delete` | Delete a dataset | - |
+| `data` | `add` or `remove` a new image to an existing dataset by its file id, or `add` or `remove` a group of images by specifying a filter | `ids`, `filter` |
+| `export` | Download all the data from a dataset | - |
+| `--help` | Return help | - |
+
+##### Positional arguments
+
+<!-- prettier-ignore -->
+| Argument | Description |
+| -------- | ----------- |
+| `filter` | `add` or `delete` images from a dataset using a filter. See [Using the `filter` argument)](#using-the-filter-argument).|
+| `ids` | `add` or `delete` images from a dataset by specifying one or more file ids as a comma-separated list. See [Using the `ids` argument)](#using-the-ids-argument).|
+| `--help` | Return help |
+
+##### Named arguments
+
+<!-- prettier-ignore -->
+| Argument | Description | Applicable commands | Required? |
+| -------- | ----------- | ------------------- | --------- |
+| `--dataset-id` | Dataset to add or remove images from, download, or rename. To retrieve these IDs, navigate to your dataset’s page in the [Viam app](https://app.viam.com), click **…** in the left-hand menu, and click **Copy dataset ID** | `rename`, `delete`, `data`, `export` | **Required** |
+| `--dataset-ids` | Dataset IDs of datasets to be listed. To retrieve these IDs, navigate to your dataset’s page in the [Viam app](https://app.viam.com), click **…** in the left-hand menu, and click **Copy dataset ID** | `list` | Optional |
+| `--destination` | Output directory for downloaded data | `export` | **Required** |
+| `--end` | ISO-8601 timestamp indicating the end of the interval | `data` | Optional |
+| `--file-ids` | File-ids to add or remove from a dataset | `data`, `export` | **Required** |
+| `--include-jsonl` | Set to `true` to include JSON Lines files for local testing |`export`| Optional |
+| `--location-id` | Location ID for the file ids being added or removed from the specified dataset (only accepts one location id) | `data` | **Required** |
+| `--name` | The name of the dataset to create or rename | `create`, `rename` | **Required** |
+| `--org-id` | Organization ID in which to edit the dataset | `create`, `data`, `list` | **Required** |
+| `--parallel` | Number of download requests to make in parallel, with a default value of 100 | `export` | Optional |
+| `--start` | ISO-8601 timestamp indicating the start of the interval | `data`| Optional |
+| `--tags` | Filter by specified tag (accepts comma-separated list) | `data` | Optional |
```

Comment:
```suggestion
| `--start` | ISO-8601 timestamp indicating the start of the interval | `data add`, `data delete` | Optional |
| `--tags` | Filter by specified tag (accepts comma-separated list) | `data add`, `data delete` | Optional |
```

Reply:
ok I wasn't sure if I should separate out or not -- will make this change everywhere then